src/common/src/Modal/ModalTopBar/ModalTopBar.tsx

```diff
@@ -19,27 +20,28 @@ const ModalTopBar = ({ hideCloseButton = false, onCloseButton, children }: Props
 			<Button
 				type="button"
 				data-testid="modal-close-button"
-				size={30}
+				size={26}
 				onClick={(evt: React.SyntheticEvent) => {
 					if (typeof onCloseButton !== 'undefined') {
 						onCloseButton(evt);
 					}
 				}}
 				variant="blank"
 			>
-				<Icon size={20}>
+				<Icon size={26}>
 					<Close />
 				</Icon>
 			</Button>
 		);
 	};
 
 	return (
-		<ModalTopBarStyle>
-			<Heading forwardedAs="div" style={{ flexGrow: 1 }}>
+		<ModalTopBarStyle data-testid="modal-top-bar" hideDivider={hideDivider}>
+			<div className="truncated" data-testid="modal-top-bar-children">
```

Comment:
Did you consider that using a class name like this does not ultimately encapsulate/protect your styles from being overridden by another component's styles?
![image](https://user-images.githubusercontent.com/1095870/179161526-1b6633a3-170b-48ae-9ca3-deb5f02f1201.png)
One could argue that creating a new styled component for this div would be a more styled-components way of doing things. I.e.:
```typescript
export const Truncated = styled.div`
    // ...
`
```

Reply:
You are right, I'm used to custom elements with a shadowroot.